src/stream-net/Reactions.cs

```diff
@@ -193,20 +229,51 @@ public async Task<Reaction> Get(string reactionID)
 
         public async Task<IEnumerable<Reaction>> Filter(ReactionFiltering filtering, ReactionPagination pagination)
         {
-            var urlPath = pagination.GetPath();
-            var request = this._client.BuildJWTAppRequest($"reaction/{urlPath}", HttpMethod.GET);
-            filtering.Apply(request);
+            //var urlPath = pagination.GetPath();
```

Comment:
these can be removed

Reply:
Removed

Reply:
Hello, may I ask when will this make its way to the master branch? I got the latest version from nuget and this is not part of it.

Reply:
i will merge it and deploy it now